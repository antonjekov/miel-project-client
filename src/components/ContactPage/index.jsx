import React from "react";
import MapContainer from "../MapContainer";
import SendMessage from "../SendMessage";
import { Row, Col } from 'react-bootstrap';


function Contact(props) {
    return (
        <Col>
        <Row>
            <Col md={{ offset: 1, span: 2 }}>
                <h3>Contact Us</h3>
                <br />
                <h4>Stara Zagora</h4>
                <br/>
                <h5>Miel Store 1</h5>
                <p>Car Simeon Veliki Str. 142<br/>tel: +359 111 111</p>
                <br/>
                <h5>Miel Store 2</h5>
                <p>General Gurko Str 47<br/>tel: +359 222 222</p>
            </Col>
        <Col md={4}>
            <MapContainer></MapContainer>
        </Col> 
        <Col md={4}>
            <SendMessage></SendMessage>
        </Col>     
           
        </Row >
        </Col>)
        
}

export default Contact