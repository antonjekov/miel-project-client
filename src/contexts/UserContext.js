import React from "react";

const UserContext = React.createContext({name: "Anton"});

export default UserContext