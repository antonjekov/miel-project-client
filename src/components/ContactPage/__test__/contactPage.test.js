import React from "react";
import ReactDOM from "react-dom"
import {render, cleanup} from "@testing-library/react"
import renderer from "react-test-renderer"
import ContactPage from "../"
import { useAuth, AuthContext } from  "../../../contexts/Auth";

afterEach(cleanup)

it('renders without crashing', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<ContactPage/>,div)
})

it('matches snapshot', ()=>{
    const tree = renderer.create(<ContactPage/>).toJSON();

    expect(tree).toMatchSnapshot();
})