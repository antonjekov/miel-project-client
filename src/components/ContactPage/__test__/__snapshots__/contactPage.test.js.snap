// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="col"
>
  <div
    className="row"
  >
    <div
      className="col-md-2 offset-md-1"
    >
      <h3>
        Contact Us
      </h3>
      <br />
      <h4>
        Stara Zagora
      </h4>
      <br />
      <h5>
        Miel Store 1
      </h5>
      <p>
        Car Simeon Veliki Str. 142
        <br />
        tel: +359 111 111
      </p>
      <br />
      <h5>
        Miel Store 2
      </h5>
      <p>
        General Gurko Str 47
        <br />
        tel: +359 222 222
      </p>
    </div>
    <div
      className="col-md-4"
    >
      <div />
      <div>
        Loading...
      </div>
    </div>
    <div
      className="col-md-4"
    >
      <form
        className=""
        onSubmit={[Function]}
      >
        <div>
          <h3>
            Send a message
          </h3>
        </div>
        <br />
        <div
          className="form-row"
        >
          <div
            className="col"
          >
            <div
              className="form-group"
            >
              <label
                className="form-label"
              >
                Name
              </label>
              <input
                className="form-control form-control"
                name="name"
                onChange={[Function]}
                placeholder="Enter your name"
                type="text"
                value=""
              />
              <div
                className="invalid-feedback"
              />
              <div
                className="valid-feedback"
              >
                Look's good
              </div>
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col"
          >
            <div
              className="form-group"
            >
              <label
                className="form-label"
              >
                Email address
              </label>
              <input
                className="form-control form-control"
                name="email"
                onChange={[Function]}
                placeholder="Enter email"
                type="email"
                value=""
              />
              <div
                className="invalid-feedback"
              />
              <div
                className="valid-feedback"
              >
                Look's good
              </div>
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <div
            className="col"
          >
            <div
              className="form-group"
            >
              <label
                className="form-label"
              >
                Message
              </label>
              <textarea
                className="form-control form-control"
                name="message"
                onChange={[Function]}
                placeholder="Write you message here"
                value=""
              />
              <div
                className="invalid-feedback"
              />
              <div
                className="valid-feedback"
              >
                Look's good
              </div>
            </div>
          </div>
        </div>
        <div
          className="form-row"
        >
          <button
            className="btn btn-warning"
            disabled={true}
            type="submit"
          >
            Send
          </button>
        </div>
        <br />
      </form>
    </div>
  </div>
</div>
`;
